commands/http/handler_test.go

```diff
@@ -69,3 +88,21 @@ func TestAllowedMethod(t *testing.T) {
 		"Access-Control-Expose-Headers":    "",
 	})
 }
+
+func TestLimitingCommands(t *testing.T) {
+	res := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "http://example.com/api/v0/cat", nil)
+
+	query := req.URL.Query()
+	query.Add("arg", "QmFooBar")
+	req.URL.RawQuery = query.Encode()
+
+	commandList := map[*commands.Command]bool{}
+
+	handler := NewHandler(commands.Context{}, corecommands.Root, "*", commandList)
```

Comment:
@travisperson another option is to define `corecommands.RootRO` inside `corecommands`. This way, no need for extra `commandList` arguments elsewhere.